.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
TODO: Add validation for required static attributes for React Component (propTypes, defaultProps)